test/frontend/components/site/siteGithubBranchesTable.test.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import SiteGithubBranchesTable from '../../../../frontend/components/site/siteGithubBranchesTable';
```

Comment:
If we drop the referenced file here in favor of the other (`githubBranchesTable`), we need to change this test to cover the other file (which isn't covered right now AFAIK).